source/data-api.txt

```diff
@@ -52,298 +73,72 @@ The Data API supports two types of endpoints:
+We recommend using the Data API when:
 
-   * - ``Document``
-     
-     - .. code-block:: json
-          :copyable: false
+- You want to run MongoDB operations from a web application or other
+  client that you can't trust.
 
-          { <content> }
-     
-     - .. code-block::
-          :copyable: false
+- You can't or don't want to manage a MongoDB driver in your server-side
+  environment. For example, some edge compute environments don't support
+  database drivers or connection pooling.
 
-          { <content> }
+- You want to integrate Atlas data access into a federated API gateway
 
-   * - ``Double``
-     
-     - .. code-block:: json
-          :copyable: false
+- You want to connect to App Services from an environment not currently
+  supported by a Realm SDK and don't want to use a driver to connect
+  over the :ref:`wire protocol <wire-protocol>`.
 
-          {
-             "$numberDouble": "10.5"
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          10.5
-   
-   * - ``Int32``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {"$numberInt":"10"}
-     
-     - .. code-block:: json
-          :copyable: false
-
-          10
-   
-   * - ``Int64``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {"$numberLong":"50"}
-     
-     - .. code-block:: json
-          :copyable: false
-
-          50
-   
-   * - ``MaxKey``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          { "$maxKey": 1 }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {}
-       
-       *No JSON equivalent*
-
-   * - ``MinKey``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          { "$minKey": 1 }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {}
-       
-       *No JSON equivalent*
-
-   * - ``ObjectId``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$oid":"5d505646cf6d4fe581014ab2"
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          "5d505646cf6d4fe581014ab2"
-
-   * - ``Regular Expression``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$regularExpression": {
-                "pattern":"^H",
-                "options":"i"
-             }
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "Pattern": "^H",
-             "Options": "i"
-          }
-
-   * - ``Timestamp``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$timestamp": {
-                "t":1565545664,
-                "i":1
-              }
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "T": 1565545664,
-             "I": 1
-          }
+An operation called through a Data API endpoint will likely take longer
```

Comment:
I would like us to be more specific here and add above the bulleted list.  Something along the lines of "Using the MongoDB Drivers to build on top of Atlas is still the general recommended approach (especially when connecting from a Server) as it will be more performant and flexible."